neurom/apps/cli.py

```diff
@@ -75,8 +75,9 @@ def view(input_file, is_3d, plane, backend, realistic_diameters):
             plot = partial(plotly_impl.plot_morph, plane=plane)
 
     plot(load_morphology(input_file))
-    if is_matplotlib and not is_3d:
-        plt.axis('equal')
+    if is_matplotlib:
+        if not is_3d:
+            plt.axis('equal')
```

Comment:
What's different here?

Reply:
coverage was complaining (but it wasn't in the previous PR...)

Reply:
You had introduce this change to fix the view cli in a previous commit. How does this change affect that issue you had before?

Reply:
remember we discussed on slack that the issue in other PR was not about python3.7 as I naively assumed, but because there was not such option for 3d plots. So this is the correct way to implement the fix on the previous PR. (we discussed after it was merged I think). It re-appeared here for some weird reason that the try: except: was not tested (but it was in previous PR), so coverage was not at 100%. The end result is exactly the same.

Reply:
ok cool. I wanted to make sure all is good.